src/main/java/seedu/duke/util/Configuration.java

```diff
@@ -0,0 +1,128 @@
+package seedu.duke.util;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+
+import seedu.duke.exceptions.ModHappyException;
+import seedu.duke.exceptions.UnkownConfigurationGroupWord;
+
+
+
+
+import static java.util.Map.entry;
+
+public class Configuration {
+
+    // Legal configuration groups.
+    public enum ConfigurationGroup {
+        COMPLETED_TASK_SHOWN;
```

Comment:
COMPLETED_TASK_ONLY or SHOW_COMPLETED_TASK_ONLY sounds better instead of COMPLETED_TASK_SHOWN.

Reply:
Also, would it be possible to map the configuration group to a number instead, so the user can just do 1=true to enable COMPLETED_TASK_ONLY?

Reply:
Good suggestion, updates the OptionParser to support both GroupWord and GroupNumber